test/numbers.jl

```diff
@@ -2771,7 +2771,8 @@ let types = (Base.BitInteger_types..., BigInt, Bool,
     for S in types
         for op in (+, -)
             if S === Float16 # type instability here?
-                @test_broken @inferred Base.promote_op(op, S)
+                # broken, fixme then remove this branch
```

Comment:
Before we had `@test_broken` the right way was commenting the test out. Now we have `@test_broken` the right way is to make `@test_broken @inferred` works.